File: GAP.Examen/GAP.Transversal/Models/Patients.cs

using System;

namespace GAP.Transversal.Models
{
    public class Patients
    {
        public Patients()
        {

        }
        public Patients(string Dni)
        {
            this.Dni = Dni;
        }
        public Patients(string Dni, string FirstName, string LastName)
        {
            this.Dni = Dni;
            this.FirstName = FirstName;
            this.LastName = LastName;
        }
        public Guid Id { get; set; }
        public string Dni { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public bool Status { get; set; }
    }
}

File: GAP.Examen/GAP.View/wwwroot/js/Patients.js
var orderTable;
$(document).ready(function () {
    $('#patients').DataTable({
        "processing": true,
        "serverSide": true,
        "filter": true,
        "orderMulti": false,
        "searching": false,
        "ajax": {
            "url": "Patient/GetPatients",
            "type": "POST",
            "datatype": "json"
        },
        "columns": [
            { "data": "dni", "name": "dni" },
            { "data": "firstName", "name": "firstName" },
            { "data": "lastName", "name": "lastName" }
        ],

        language: language
    });

});